llvm/lib/Target/AMDGPU/AMDGPUISelLowering.cpp

```diff
@@ -6043,3 +6043,11 @@ bool AMDGPUTargetLowering::isReassocProfitable(MachineRegisterInfo &MRI,
                                                Register N0, Register N1) const {
   return MRI.hasOneNonDBGUse(N0); // FIXME: handle regbanks
 }
+
+bool AMDGPUTargetLowering::hasAndNot(SDValue Op) const {
```

Comment:
Should move this to SIISelLowering. Not sure if r600 has anything relevant

Reply:
Done.